src/include/storage/block_layout.h

```diff
@@ -9,7 +9,8 @@ namespace terrier::storage {
 // Internally we use the sign bit to represent if a column is varlen or not. Down to the implementation detail though,
 // we always allocate 16 bytes for a varlen entry, with the first 8 bytes being the pointer to the value and following
 // 4 bytes be the size of the varlen. There are 4 bytes of padding for alignment purposes.
-#define VARLEN_COLUMN static_cast<uint8_t>(0x90)  // 16 with the first (most significant) bit set to 1
+#define VARLEN_COLUMN static_cast<uint16_t>(0x8010)  // 16 with the first (most significant) bit set to 1
```

Comment:
When I experimented with this 0x9000 and 0x8000 wouldn't work and only 0x8010 would. I was wondering what the second 1 is used for. From the rest of the code it seems like only the most significant bit being set to 1 is the indicator for that attribute size being a varlen.

Reply:
the remaining bits have to be 16 as explained by the comment